modules/Campaigns/Popup_picker.html

```diff
@@ -37,84 +37,92 @@
-			<input type="hidden" name="form" value="{FORM}" />
-	  </form>
-	 </tr>
-	</table>	
-  </td>
- </tr>
+    </tr>
 </table>
 
 <script type="text/javascript">
-<!--
-/* initialize the popup request from the parent */
+  <!--
+  /* initialize the popup request from the parent */
 
-if(window.document.forms['popup_query_form'].request_data.value == "")
-{
-	window.document.forms['popup_query_form'].request_data.value
-		= JSON.stringify(window.opener.get_popup_request_data());
-}
--->
+  if (window.document.forms['popup_query_form'].request_data.value == "") {
+    window.document.forms['popup_query_form'].request_data.value
+      = JSON.stringify(window.opener.get_popup_request_data());
+  }
+  -->
 </script>
 <!-- END: SearchHeader -->
 
 <table width="100%" border="0" cellspacing="0" cellpadding="0" class="list view">
-<!-- BEGIN: list_nav_row -->
-{PAGINATION}
-<!-- END: list_nav_row -->
+    <!-- BEGIN: list_nav_row -->
+    {PAGINATION}
+    <!-- END: list_nav_row -->
 
- <tr height="20" >	
-	<td scope="col" width="5%"   nowrap>{CHECKALL}</td>
- 	<td scope="col" width="35%" ><span><a href="{ORDER_BY}name" class="listViewThLinkS1">{MOD.LBL_CAMPAIGN_NAME}{arrow_start}{name_arrow}{arrow_end}</a></span></td>
-	<td scope="col"  width="15%"  NOWRAP>&nbsp;<span><a href="{ORDER_BY}status" class="listViewThLinkS1">{MOD.LBL_LIST_STATUS}{arrow_start}{status_arrow}{arrow_end}</a></span></td>
-	<td scope="col"  width="15%"  NOWRAP>&nbsp;<span><a href="{ORDER_BY}campaign_type" class="listViewThLinkS1">{MOD.LBL_LIST_TYPE}{arrow_start}{campaign_type_arrow}{arrow_end}</a></span></td>
- </tr>
+    <tr height="20">
+        <td scope="col" width="5%" nowrap>{CHECKALL}</td>
+        <td scope="col" width="35%">
+            <slot><a href="{ORDER_BY}name" class="listViewThLinkS1">{MOD.LBL_CAMPAIGN_NAME}{arrow_start}{name_arrow}{arrow_end}</a>
+            </slot>
+        </td>
+        <td scope="col" width="15%" NOWRAP>&nbsp;<slot><a href="{ORDER_BY}status" class="listViewThLinkS1">{MOD.LBL_LIST_STATUS}{arrow_start}{status_arrow}{arrow_end}</a>
+        </slot>
+        </td>
+        <td scope="col" width="15%" NOWRAP>&nbsp;<slot><a href="{ORDER_BY}campaign_type" class="listViewThLinkS1">{MOD.LBL_LIST_TYPE}{arrow_start}{campaign_type_arrow}{arrow_end}</a>
+        </slot>
+        </td>
+    </tr>
 
-<!-- BEGIN: row -->
- <tr height="20"  class="{ROW_COLOR}S1">
-	<td valign="top">{PREROW}</td>
-	<td scope='row' valign=TOP><span><{TAG_TYPE} href="#"  onclick="send_back('Campaigns','{CAMPAIGN.ID}');">{CAMPAIGN.NAME}</{TAG_TYPE}></span></td>
-    <td valign=TOP><span>{CAMPAIGN.STATUS}</span></td>
-    <td valign=TOP><span>{CAMPAIGN.CAMPAIGN_TYPE}</span></td>
+    <!-- BEGIN: row -->
+    <tr height="20" class="{ROW_COLOR}S1">
+        <td valign="top">{PREROW}</td>
+        <td scope='row' valign=TOP>
+            <slot><{TAG_TYPE} href="#" onclick="send_back('Campaigns','{CAMPAIGN.ID}');">{CAMPAIGN.NAME}</
+            {TAG_TYPE}></slot></td>
```

Comment:
Hi @Dillon-Brown, here maybe we should move the `{TAG_TYPE}>` to the line above, and keep the `</slot>` on this line

Reply:
Pushed!